src/target/riscv/riscv.c

```diff
@@ -4800,25 +4802,11 @@ static const struct command_registration riscv_exec_command_handlers[] = {
 			"The final ebreak instruction is added automatically, if needed."
 	},
 	{
-		.name = "set_enable_eq_match_trigger",
-		.handler = riscv_set_enable_eq_match_trigger,
-		.mode = COMMAND_CONFIG,
-		.usage = "[on|off]",
-		.help = "When on, allow OpenOCD to use equality match trigger in wp."
-	},
-	{
-		.name = "set_enable_napot_trigger",
-		.handler = riscv_set_enable_napot_trigger,
-		.mode = COMMAND_CONFIG,
-		.usage = "[on|off]",
-		.help = "When on, allow OpenOCD to use NAPOT trigger in wp."
-	},
-	{
-		.name = "set_enable_ge_lt_trigger",
-		.handler = riscv_set_enable_ge_lt_trigger,
-		.mode = COMMAND_CONFIG,
-		.usage = "[on|off]",
-		.help = "When on, allow OpenOCD to use GE/LT triggers in wp."
+		.name = "set_enable_trigger_type",
+		.handler = riscv_set_enable_trigger_type,
+		.mode = COMMAND_ANY,
+		.usage = "[('eq'|'napot'|'ge_lt') ('wp'|'none')]",
```

Comment:
The option `all` should also be added to the `.usage`.

Reply:
Addressed.